vaadin-grid-flow/src/main/java/com/vaadin/flow/component/grid/Grid.java

```diff
@@ -2299,7 +2301,13 @@ private int getLastFooterLayerIndex() {
         return -1;
     }
 
+    /**
+     * @inheritDocs
+     * @deprecated use instead one of the setDataSource methods from
+     * {@link HasListDataView} or {@link HasLazyDataView}
```

Comment:
```suggestion
     * provide access to either {@link GridListDataView} or
     * {@link GridLazyDataView}..
```

Reply:
Done, but can't link to GridLazyDataView here as it doesn't exist.